tests/transforms/test_classical_jacobian.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
```suggestion
# limitations under the License.
"""Tests for the qml.transforms.classical_jacobian function."""
```
(simple docstring to help developers easily work out the context of the file)

Reply:
Thanks!